src/bikeracks/templates/scripts.js

```diff
@@ -366,13 +360,14 @@ BikeMap.prototype.createMarker = function(state) {
     else {
         // we didn't find a marker, make a new one
         marker = L.marker([state.latitude, state.longitude], {uniqueId: state.rack_id}); 
-        
+        // Figure out marker's status
+        let status = state.upvote_count > state.downvote_count;
```

Comment:
Now that this is appearing all over the place, it should probably be a method on `state` objects or even a function that just takes `state` as an argument. This also allows you to change the meaning of approval more easily if you decide that 50/50 isn't good enough, for example.

```js
if (state.isApproved()) {
  this.allApproved.addLayer(marker);
} else {
  this.allNotApproved.addLayer(marker);
}
```

Reply:
Done.